runtime/src/main/java/org/corfudb/runtime/view/AddressSpaceView.java

```diff
@@ -557,6 +558,21 @@ public void invalidateClientCache() {
         return result;
     }
 
+    /**
+     * Fetch a collection of addresses for insertion into the cache.
+     * The fetch results are <b>NOT</b> cached on log unit server.
+     * The result map returned is ordered by address.
+     *
+     * @param addresses    collection of addresses to read from.
+     * @param waitForWrite flag whether wait for write is required or hole fill directly.
+     * @return an ordered map of read addresses.
+     */
+    @Nonnull
```

Comment:
looks like you don't need this annotation here

Reply:
Done.